test/api/test_dataset_collections.py

```diff
@@ -97,11 +97,9 @@ def test_create_list_of_new_pairs(self):
 
     def test_list_download(self):
         dataset_collection = self.dataset_collection_populator.create_list_in_history(self.history_id).json()
+        self.dataset_collection_populator.wait_for_dataset_collection(dataset_collection, assert_ok=True)
         returned_dce = dataset_collection["elements"]
-        assert len(returned_dce) == 3, dataset_collection
-        returned_datasets = [dce["object"] for dce in returned_dce]
-        for element in returned_datasets:
-            self.dataset_populator.wait_for_dataset(history_id=self.history_id, dataset_id=element['id'], assert_ok=True)
+        assert len(dataset_collection["elements"]) == 3, dataset_collection
```

Comment:
You can probably revert the change to this line to:
```
assert len(returned_dce) == 3, dataset_collection
```
for consistency with the other tests.

Reply:
Right, will do!